nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -175,7 +175,11 @@ def setup(self):
                 return False
 
             if self.args.gpu:
-                gpus = self.args.gpu.split(",")
+                gpus, success = self.split_gpus(self.args.gpu)
+                if not success:
+                    self.logger.error(f"GPUs group list in wrong format. Please use format like: [0,1],[1,2] ...")
```

Comment:
need to change to `self.logger.error("GPUs group list in wrong format. Please use format like: [0,1],[1,2] ...")`
otherwise style check will fail